api/controller/migrationtarget/client.go

```diff
@@ -158,36 +158,6 @@ func (c *Client) UploadTools(ctx context.Context, modelUUID string, r io.Reader,
 
 // UploadResource uploads a resource to the migration endpoint.
 func (c *Client) UploadResource(ctx context.Context, modelUUID string, res resource.Resource, r io.Reader) error {
-	args := makeResourceArgs(res)
-	args.Add("application", res.ApplicationName)
-	err := c.resourcePost(ctx, modelUUID, args, r)
-	return errors.Trace(err)
-}
-
-// SetPlaceholderResource sets the metadata for a placeholder resource.
-func (c *Client) SetPlaceholderResource(ctx context.Context, modelUUID string, res resource.Resource) error {
-	args := makeResourceArgs(res)
-	args.Add("application", res.ApplicationName)
-	err := c.resourcePost(ctx, modelUUID, args, nil)
-	return errors.Trace(err)
-}
-
-// SetUnitResource sets the metadata for a particular unit resource.
-func (c *Client) SetUnitResource(ctx context.Context, modelUUID, unit string, res resource.Resource) error {
-	args := makeResourceArgs(res)
-	args.Add("unit", unit)
-	err := c.resourcePost(ctx, modelUUID, args, nil)
-	return errors.Trace(err)
-}
-
-func (c *Client) resourcePost(ctx context.Context, modelUUID string, args url.Values, r io.Reader) error {
-	uri := "/migrate/resources?" + args.Encode()
-	contentType := "application/octet-stream"
-	err := c.httpPost(ctx, modelUUID, r, uri, contentType, nil, nil)
-	return errors.Trace(err)
-}
-
-func makeResourceArgs(res resource.Resource) url.Values {
 	args := url.Values{}
 	args.Add("name", res.Name)
 	args.Add("type", res.Type.String())
```

Comment:
question: Why still send the data which is no longer used? E.g. `type`, `description` et al

Reply:
Good point, have removed path and description, the rest are still needed though.